modelling/src/test/java/org/axonframework/modelling/command/AnnotationCommandTargetResolverTest.java

```diff
@@ -16,159 +16,282 @@
 
 package org.axonframework.modelling.command;
 
-import org.junit.Before;
-import org.junit.Test;
-
-import java.util.UUID;
-
 import static org.axonframework.commandhandling.GenericCommandMessage.asCommandMessage;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 
+import java.lang.annotation.ElementType;
```

Comment:
I don't think the import switch is necessary.

Reply:
My tooling. I’ll try to fix it, but it isn’t sadly that easy:-)